Source/RunActivity/Viewer3D/Popups/WindowControls.cs

```diff
@@ -115,6 +115,9 @@ public enum LabelAlignment
         Left,
         Center,
         Right,
+        Cab3DLeft,
+        Cab3DCenter,
+        Cab3DRight,
```

Comment:
This change is not okay; this `enum` is for the popup windows and adding values that are **not** used by the popup windows is confusing and wrong – `CabViewDigitalRenderer` should be using its own `enum` if you want additional/different `enum` values

Reply:
I will put in a comment the explanation that I have written here http://www.elvastower.com/forums/index.php?/topic/35498-justification-alignment-for-digitals-in-3dcabs/page__view__findpost__p__275959 : as of now existing 3D cabs have left-aligned digitals independently from the value of Justification. If I now respect the value of Justification,, existing 3D cabs will change the alignment of their digitals, which is not desirable.

Reply:
This response does not match the issue here. You **cannot** add these three values to this particular `enum`. It is completely and utterly the wrong thing to do here.

If you absolutely must keep track of 6 different alignment values, please create a _new_ `enum` in the cab view code.